### PR TITLE
Feature/f п.4 shared layout

### PR DESCRIPTION
**Pull request by SerhiiOnyshchenko**
В комітах все описано

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -18,7 +19,15 @@ export default function App() {
...
-        <Route exact path="/" element={<PrivateRoute></PrivateRoute>} />
+        <Route
+          exact
+          path="/"
+          element={
+            <PrivateRoute>
+              <HomePage />
+            </PrivateRoute>
+          }
+        />
```
Comment (ihorshark) [inline src/App.jsx]: I think there's no need to use exact attribute anymore. It was useful with Switch construction.

File: src/pages/HomePage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import Container from 'components/Container';
+import s from './index.module.css';
+
+export default function HomePage() {
+  return (
+    <div className={s.HomePage}>
+      <Container>
+        <h1 className={s.Title}>Take good care of your small pets</h1>
+      </Container>
+    </div>
```
Comment (ihorshark) [inline src/pages/HomePage/index.jsx:11]: If we take care of semantics, it would be better to change div to section tag. But this is not essential.

**Review by ihorshark — COMMENTED** [review]
In my opinion it is a good work for a starting repo. Just a little notice - PablicRoute folder can be renamed to PublicRoute, just a small spelling mistake.

File: src/App.jsx (hunk 1 above)
Comment (SerhiiOnyshchenko) [reply src/App.jsx]: I agree with your opinion.

File: src/pages/HomePage/index.jsx (hunk 2 above)
Comment (SerhiiOnyshchenko) [reply src/pages/HomePage/index.jsx:11]: It is not need, because here will be only one div. It's not landing page.

**Comment by SerhiiOnyshchenko** [comment]
Thank you for a spelling mistake.

**Review by ihorshark — APPROVED** [review]

File: src/pages/HomePage/index.jsx (hunk 2 above)
Comment (ihorshark) [reply src/pages/HomePage/index.jsx:11]: Got it.

**Pull request merged**